compose/README.md

```diff
@@ -79,6 +80,29 @@ container is using the newest image, e.g.:
 | clamavd                                 | `tcp/3310`     | `tcp/62006` |
 | nginx » archivematica-dashboard         | `tcp/80`       | `tcp/62080` |
 | nginx » archivematica-storage-service   | `tcp/8000`     | `tcp/62081` |
+| selenium-hub                            | `tcp/4444`     | `tcp/62100` |
+
+### Acceptance tests
+
+There is an extra Compose file called `docker-compose.acceptance-tests.yml` that defines some extra services to run our acceptance tests, e.g. Selenium Standalone Server, Selenium Chrome Node or Selenium Firefox Node. In order to run the tests use the following command:
+
+    $ make at-run
```

Comment:
I need to support passing arguments to behave, e.g.:

    $ make ARGS="--tags=@wip" at-run